press/legacy_publishing/litezip.py

```diff
@@ -34,13 +34,6 @@ def publish_litezip(struct, submission, db_conn, coll_changes_allowed=True):
     except IndexError:  # pragma: no cover
         raise NotImplementedError('litezip without collection')
 
-    try:
```

Comment:
I have no idea how this code got into master previously. It's clearly bogus, so I'm just deleting it here which would fix issue #308

Reply:
This is to allow for collection structure only publications. That is if you send a litezip that only contains a `collection.xml`.